src/sim_engine/filesystem.rs

```diff
@@ -6,15 +6,29 @@ use engine::Filesystem;
 
 #[derive(Debug, Clone)]
 pub struct SimFilesystem {
+    pub name: String,
```

Comment:
we've got a "name" field in the Pool already... we should either remove or keep both, right?